File: fizzBuzz.pl
#!/usr/bin/env perl

use strict; 
use warnings;

for (my $i = 1; $i <= 100; $i++)
{
    if ($i % 15 == 0) {
        print "FizzBuzz"
    }
    elsif ($i % 3 == 0) {
        print "Fizz";
    }
    elsif ($i % 5 == 0) {
        print "Buzz";
    }
    else {
        print $i;
    }

    print "\n";
}

File: fizzBuzz.rb
#!/usr/bin/env ruby

1.upto(100) do |i|
	msg = String.new #inefficient! 
	msg += "Fizz" if i % 3 == 0	
	msg += "Buzz" if i % 5 == 0
	puts (msg == "" ? i : msg)
end
